src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -1,16 +1,30 @@
 from enum import Enum
 from typing import Optional
 
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID, extract_use_case_id
+
 
 class AggregationOption(Enum):
     HIST = "hist"
+    TEN_SECOND = "ten_second"
```

Comment:
I guess the same thing as https://github.com/getsentry/snuba/pull/4747#discussion_r1337603493 applies here, 10 seconds shouldn't be a special case